lightning/src/ln/functional_test_utils.rs

```diff
@@ -453,17 +453,33 @@ impl<'a, 'b, 'c> Node<'a, 'b, 'c> {
 	/// several of the signing methods. Currently, only `get_per_commitment_point` and
 	/// `release_commitment_secret` are affected by this setting.
 	#[cfg(test)]
-	pub fn set_channel_signer_available(&self, peer_id: &PublicKey, chan_id: &ChannelId, available: bool) {
+	pub fn set_channel_signer_available(&self, peer_id: &PublicKey, chan_id: &ChannelId, funding_txo: Option<OutPoint>, available: bool) {
```

Comment:
Kind of a minor nit, but... do you think this would be cleaner if you separated the monitor stuff out as a separate method? E.g., `set_monitor_signer_available`? I.e. like

```
#[cfg(test)]
pub fn set_monitor_signer_available(&self, funding_txo: OutPoint, available: bool) { ... }
```